rules/prefer-flat-map.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const MESSAGE_ID = 'preferFlatMap';
+
+const isFlat = node => {
```

Comment:
```suggestion
const isFlatMethod = node => {
```

Same with the below.